test/e2e/test_job_submissions.py

```diff
@@ -1927,19 +1920,33 @@ def test_worker_job_attachments_no_outputs_does_not_fail_job(
         [
             "COPIED",
             # Worker E2E test doesn't run VFS, but this helps verify VIRTUAL fallback and job run successfully
-            "VIRTUAL",
+            # "VIRTUAL",
         ],
     )
     def test_worker_job_attachments_dep_data_flow_linux(
         self,
         deadline_resources: DeadlineResources,
         deadline_client: DeadlineClient,
         session_worker: EC2InstanceWorker,
+        worker_config: DeadlineWorkerConfiguration,
         file_system: str,
     ) -> None:
         job_bundle_path: str = os.path.join(
             os.path.dirname(__file__), "job_attachment_bundle", "dep_data_flow", "linux_bundle"
         )
+        if worker_config.worker_env_var:
+            job_parameters: List[Dict[str, str]] = [
+                {
+                    "name": "AssetSync",
+                    "value": get_feature_value(
+                        worker_env_var=worker_config.worker_env_var,
+                        feature_name="ASSET_SYNC_JOB_USER_FEATURE",
+                        default_value="False",
```

Comment:
What about this to make it explicit?

```suggestion
                    "value": worker_env_var.get("ASSET_SYNC_JOB_USER_FEATURE", "False")
```

Reply:
as mention above, will fix in new rev!